packages/litexa/src/parser/litexa.pegjs

```diff
@@ -1539,10 +1539,14 @@ SlotTypeStatement
   }
 
 /* litexa [otherwise]
-Defines a catch all intent that will be executed should its parent state receive
-an intent that it has no explicit handler for. It's always possible for the user
-to say something unexpected, so you should generally always have something to say
-to guide them back on track in these cases.
+Defines a handler that will catch any skill intent that is not explicitly handled
+by the parent state (or the global state). `otherwise` is a great catch-all
+for anything unexpected the user might say:
+
+1. an intent that is supported elsewhere in the skill but makes no sense in the parent state
+2. an utterance that isn't supported by the skill at all, which could arrive as an
+[AMAZON.FallbackIntent](https://developer.amazon.com/blogs/alexa/post/c97f3bb7-9701-41e8-ac06-a3a44b9f1638/use-the-new-fallback-intent-to-respond-gracefully-to-unexpected-customer-requests)
+if supported by the locale the skill is running in.
```

Comment:
The word support here is a bit confusing; you still need to define the fallback intent in your skill model for it to exist, even if you aren't going to handle it explicitly in your skill (declared in an unreachable state).

Would also change the link to technical docs: https://developer.amazon.com/en-US/docs/alexa/custom-skills/standard-built-in-intents.html#fallback

Reply:
Take 2: Let me know what you think. Removed the `FallbackIntent` again as that might be informational overload here.